x-pack/plugins/spaces/tsconfig.json

```diff
@@ -50,7 +50,8 @@
     "@kbn/core-notifications-browser",
     "@kbn/logging",
     "@kbn/core-logging-browser-mocks",
-    "@kbn/core-http-router-server-mocks"
+    "@kbn/core-http-router-server-mocks",
+    "@kbn/security-plugin"
```

Comment:
This might trigger a cyclic dependency warning, since the security plugin also depends on the spaces plugin